draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -4477,7 +4539,7 @@ RESPONSE:
             "generic-trigger-extension-value": {
               "unix-time-window": {
                 "start": 946717200,
-                "end": 946746000
+                "end"  : 946746000
```

Comment:
While I don't care too much about the spaces between the " and the :, this change now makes this inconsistent with how other sections format "start" and "end".